x-pack/plugins/observability_solution/infra/common/formatters/snapshot_metric_formats.ts

```diff
@@ -29,7 +29,7 @@ export const METRIC_FORMATTERS: MetricFormatters = {
     formatter: InfraFormatterType.percent,
     template: '{{value}}',
   },
-  ['cpuTotal']: {
+  ['cpuV2']: {
```

Comment:
Awesome, I was about to open an issue to change this for consistency, thanks for doing it